app/helpers/court_dates_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Helper methods for court_dates
+module CourtDatesHelper
+  def when_do_we_have_court_dates(casa_case)
+    date_query = casa_case.court_dates.includes(:hearing_type).ordered_ascending
+    last_court_date = date_query.last
+    first_court_date = date_query.first
```

Comment:
**suggestion:** it feels to me like you could use `date_query.last.date` directly instead of storing this in a variable. It also seems to me like it doesn't lose in meaning/how-understandable-it-is. WDYT?